app/src/app/[locale]/page.tsx

```diff
@@ -0,0 +1,25 @@
+import { Metadata } from "next";
+import { isFeatureEnabled } from "src/services/feature-flags";
+
+import { getTranslations } from "next-intl/server";
+
+import { View } from "./view";
+
+interface RouteParams {
+  locale: string;
+}
+
+export async function generateMetadata({ params }: { params: RouteParams }) {
```

Comment:
Maybe overkill, but do you think worth linking to this section of the docs? https://nextjs.org/docs/app/api-reference/functions/generate-metadata

I imagine we don't want to reference for everything, but was wondering if for things that are new for server components (which many folks may be less familiar with at first) it could help 🤷‍♀️

Reply:
Good question. I have a follow-up ticket where I want to add more docs so will fold code comments into that scope since I think it relates. https://github.com/navapbc/template-application-nextjs/issues/262